frontend/__tests__/containers/ReportStep/DoraMetrics.test.tsx

```diff
@@ -99,4 +101,21 @@ describe('Report Card', () => {
 
     expect(screen.queryByLabelText('dora metrics dialog')).not.toBeInTheDocument();
   });
+
+  it('should show four new labels in the dora metrics when version is less than 1.3.0', () => {
+    setup();
+
+    const newLabels = screen.queryAllByLabelText('new label');
+
+    expect(newLabels.length).toEqual(4);
+    newLabels.forEach((it) => expect(it).toBeInTheDocument());
+  });
+
+  it('should not show any new label in the dora metrics when version is more than 1.3.0', () => {
```

Comment:
how to compare `1.0.0-alpha、1.0.0-alpha.1、1.0.0-0.3.7、1.0.0-x.7.z.92。`

ref to https://semver.org/lang/zh-CN/

Reply:
done